### DISCUSSION TITLE
Codespaces don't start up

### DISCUSSION THREAD

**Discussion by maiertech** [community/community#24982] [Codespaces]
<p>Today I keep seeing this in VS Code (not going away)</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/e/2edfb4ce1d958cde92b6ef3c4c6cf5ed3eca07e6.png" data-download-href="/uploads/short-url/6GFfD7MJGMwEambGmdKDscUpVhI.png?dl=1" title="Screen Shot 2021-02-10 at 10.29.47 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1482402/181087998-9a1aa853-1b17-4a38-8e04-89b3c72444f2.png" alt="Screen Shot 2021-02-10 at 10.29.47 AM" data-base62-sha1="6GFfD7MJGMwEambGmdKDscUpVhI" width="689" height="199" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/e/2edfb4ce1d958cde92b6ef3c4c6cf5ed3eca07e6_2_689x199.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/e/2edfb4ce1d958cde92b6ef3c4c6cf5ed3eca07e6_2_1033x298.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/e/2edfb4ce1d958cde92b6ef3c4c6cf5ed3eca07e6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/e/2edfb4ce1d958cde92b6ef3c4c6cf5ed3eca07e6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-02-10 at 10.29.47 AM</span><span class="informations">1196×346 11.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>or this in the browser (also not going away)</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/b/7bdfd0ad2805233f4fb6c8cd293711b60a609407.jpeg" data-download-href="/uploads/short-url/hFQfxhZkipLaKAAwbmqoO4PMLYj.jpeg?dl=1" title="Screen Shot 2021-02-10 at 10.33.05 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1482402/181087999-cae9e0df-3685-4103-ad92-3b1c0983dbec.jpeg" alt="Screen Shot 2021-02-10 at 10.33.05 AM" data-base62-sha1="hFQfxhZkipLaKAAwbmqoO4PMLYj" width="690" height="309" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7bdfd0ad2805233f4fb6c8cd293711b60a609407_2_690x309.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7bdfd0ad2805233f4fb6c8cd293711b60a609407_2_1035x463.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7bdfd0ad2805233f4fb6c8cd293711b60a609407_2_1380x618.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/b/7bdfd0ad2805233f4fb6c8cd293711b60a609407_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-02-10 at 10.33.05 AM</span><span class="informations">1380×618 49.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Are you aware of any technical glitches?</p>

**Comment by maiertech** [comment]
<p>Launching new codespaces seems to work fine, but launching existing ones fails. Solved by disposing of old codespaces and create new ones.</p>

**Comment by don1989** [comment]
<p>This is also happening for us. We get the error in VS Code with the following stack:</p>
<pre><code class="lang-auto">[API I] Successfully signed in
[RemoteExplorer I] Updating codespace details
[RemoteExplorer I] Updating codespace performance info
[RemoteResolver I] Resolving: xyz-ui-v23g with params undefined
[RemoteResolver I] Fetching codespace details by name
[RemoteExplorer I] Updating codespace details
[RemoteExplorer I] Updating codespace performance info
[RemoteResolver I] Successfully fetched codespace details
[RemoteExplorer I] Updating codespace details
[RemoteExplorer I] Updating codespace performance info
[RemoteResolver I] Attempting to connect to codespace
[RemoteResolver I] Progress: Connecting to codespace...
[RemoteExplorer I] Updating codespace details
[RemoteExplorer I] Updating codespace performance info
[WorkspaceClient:WorkspaceApi I] getWorkspaceSessionInfo -&gt; workspaceId:undefined
[WorkspaceServices E] Exception joining workspace undefined&gt; object #0: Cannot read property 'toUpperCase' of undefined
TypeError: Cannot read property 'toUpperCase' of undefined
    at WorkspaceApi.getWorkspaceAccess (c:\Users\xyz\.vscode\extensions\workspace\core\dist\src\WorkspaceApi.js:61:1)
    at Function.getWorkspaceSessionInfo (c:\Users\xyz\.vscode\extensions\workspace\core\dist\src\WorkspaceApi.js:43:1)
    at WorkspaceClient.getSessionInfo (c:\Users\xyz\.vscode\extensions\workspace\client\dist\src\WorkspaceClient.js:113:1)
    at WorkspaceClient.join (c:\Users\xyz\.vscode\extensions\workspace\client\dist\src\WorkspaceClient.js:46:1)
    at WorkspaceServices.join (c:\Users\xyz\.vscode\extensions\github.codespaces-0.7.1\src\workspaceServices.ts:110:48)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at CodespacesResolver.connectToWorkspace (c:\Users\xyz\.vscode\extensions\github.codespaces-0.7.1\src\codespacesResolver.ts:588:9)
    at c:\Users\xyz\.vscode\extensions\github.codespaces-0.7.1\src\codespacesResolver.ts:254:21
[RemoteResolver I] Progress: Starting remote server...
</code></pre>

**Comment by jkeech** [comment, marked as answer]
<p>We had an issue in the West Europe region where resume requests were not being processed. We’ve resolved the issue, so you should be able to resume now. Please let us know if you continue to see issues connecting to your codespaces. Thanks!</p>
